plugin/evm/vm_upgrade_bytes_test.go

```diff
@@ -210,7 +211,7 @@ func TestNetworkUpgradesOverridden(t *testing.T) {
 	// verify upgrade overrides
 	require.False(t, restartedVM.chainConfigExtra().IsGranite(uint64(upgrade.InitiallyActiveTime.Unix())))
 	require.True(t, restartedVM.chainConfigExtra().IsGranite(graniteTimestamp))
-	require.False(t, restartedVM.currentRules().IsGranite)
+	require.False(t, currentRules(tvm.vm).IsGranite)
```

Comment:
This seems to have changed which vm we are getting the rules from.
```suggestion
	require.False(t, currentRules(restartedVM).IsGranite)
```